sklearn/inspection/_plot_estimators.py

```diff
@@ -0,0 +1,263 @@
+from collections import namedtuple
+from contextlib import closing
+from io import StringIO
+
+from .._config import config_context
+from ..base import BaseEstimator
+from ..pipeline import Pipeline
+from ..pipeline import FeatureUnion
+from ..compose import ColumnTransformer
+from ..ensemble import VotingClassifier, VotingRegressor
+
+
+def _estimator_tool_tip(estimator):
+    """Replace newlines to allow for css content: attr(...) to properly
+    display tooltips.
+    """
+    return str(estimator).replace('\n', '&#xa;')
+
+
+def _write_label_html(out, name, tool_tip):
+    """Write label to html"""
+    out.write('<div class="sk-label" sk-data-tooltip="{}">'
+              '{}</div>'.format(tool_tip, name))
+
+
+_EstHTMLInfo = namedtuple('_EstHTMLInfo',
+                          'type, estimators, names, name_tips')
+
+
+def _type_of_html_estimator(estimator):
+    """Generate information about how to display an estimator.
+    """
+    if isinstance(estimator, str):
+        return _EstHTMLInfo('single', estimator, estimator, estimator)
+
+    elif estimator is None:
+        return _EstHTMLInfo('single', estimator, 'None', 'None')
+
+    elif isinstance(estimator, Pipeline):
+        estimators = [step[1] for step in estimator.steps]
+        names = [step[0] for step in estimator.steps]
+        name_tips = [_estimator_tool_tip(est) for est in estimators]
+        return _EstHTMLInfo('serial', estimators, names, name_tips)
+
+    elif isinstance(estimator, ColumnTransformer):
+        estimators = [trans[1] for trans in estimator.transformers]
+        names = [trans[0] for trans in estimator.transformers]
+        name_tips = [trans[2] for trans in estimator.transformers]
+        return _EstHTMLInfo('parallel', estimators, names, name_tips)
+
+    elif isinstance(estimator, FeatureUnion):
+        estimators = [trans[1] for trans in estimator.transformer_list]
+        names = [trans[0] for trans in estimator.transformer_list]
+        name_tips = [_estimator_tool_tip(est) for est in estimators]
+        return _EstHTMLInfo('parallel', estimators, names, name_tips)
+
+    elif isinstance(estimator, (VotingClassifier, VotingRegressor)):
+        estimators = [est[1] for est in estimator.estimators]
+        names = [est[0] for est in estimator.estimators]
+        name_tips = [_estimator_tool_tip(est) for est in estimators]
+        return _EstHTMLInfo('parallel', estimators, names, name_tips)
```

Comment:
I would check if it has a ``base_estimator_`` or ``estimator_`` and then iterate into it.